en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -48,21 +49,21 @@ Follow the procedure below to create a revision of the API and deploy it:
 Now you are ready to test the deployed API.
 
 ## Step 3: Test the API
-Follow the procedure below to test the API revision that you deployed:
+Follow this procedure to test the API revision that you deployed:
 
 1. Click **Test** in the left pane.
-2. Click on the `GET` resource you added to expand it.
+2. Click the `GET` resource you added to expand it.
 
     ![API GET resource](../assets/img/apis/api-resource.png){.cInlineImage-full}
 
-3. Click **Try it out**, and then click **Execute**. You can see `{"hello": "world"}` as the response from the API.
+3. Click **Try it out** and then click **Execute**. You can see `{"hello": "world"}` as the response from the API.
 
     ![Response received from the service execution](../assets/img/apis/response.png){.cInlineImage-full}
 
-Now you are ready to publish the API to the Choreo Devportal so that external consumers can consume the API.
+Now you are ready to publish the API to the Choreo Developer Portal so that external consumers can consume the API.
 
 ## Step 4: Publish the API
-Follow the procedure below to publish the API and view the published API on the Choreo Devportal:
+Follow this procedure to publish the API and view the published API on the Choreo Developer Portal:
```

Comment:
Shouldn't we say `Choreo API Developer Portal` instead of `Choreo Developer Portal`? Otherwise, this is bit confusing with the Developer Console.

Reply:
Related issue https://github.com/wso2-enterprise/choreo/issues/4891

Reply:
Renamed `Choreo Developer Portal` to `Choreo API Developer Portal`.
I'll rename the button once the change is done in the UI.